packages/cli/src/services/abstract-check-runner.ts

```diff
@@ -136,22 +144,9 @@ export default abstract class AbstractCheckRunner extends EventEmitter {
         result.checkRunData = await assets.getCheckRunData(region, checkRunDataPath)
       }
 
-      try {
-        if (result.hasFailures) {
-          if (playwrightTestTraceFiles && playwrightTestTraceFiles.length) {
-            result.traceFilesUrls = await Promise
-              .all(playwrightTestTraceFiles.map((t: string) => assets.getAssetsLink(region, t)))
-          }
-          if (playwrightTestVideoFiles && playwrightTestVideoFiles.length) {
-            result.videoFilesUrls = await Promise
-              .all(playwrightTestVideoFiles.map((t: string) => assets.getAssetsLink(region, t)))
-          }
-        }
-      } catch {
-        // TODO: remove the try/catch after deploy endpoint to fetch assets link
-      }
+      const links = testResultId && result.hasFailures && await this.getShortLinks(testResultId)
```

Comment:
More of a product question - should we also set a shortlink if the `verbose` flag is passed? I think that that could be useful. It'll start creating many shortlinks, though.

Reply:
I asked Tim in Slack. Is it straightforward to also check the `verbose` flag here as well? We could also come back to this in a later PR.